src/RoomNotifs.js

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import MatrixClientPeg from './MatrixClientPeg';
+import PushProcessor from 'matrix-js-sdk/lib/pushprocessor';
+import q from 'q';
+
+export function getRoomNotifsState(roomId) {
+    // look through the override rules for a rule affecting this room:
+    // if one exists, it will take precedence.
+    const muteRule = findOverrideMuteRule(roomId);
+    if (muteRule && muteRule.enabled) {
```

Comment:
should the check of `enabled` be done in `findOverrideMuteRule`? what if there are multiple rules which look like mute rules, of which the first is disabled and the second enabled?